tests/fixtures/saas/adyen_fixtures.py

```diff
@@ -0,0 +1,79 @@
+from typing import Any, Dict, Generator
+
+import pydash
+import pytest
+
+from tests.ops.integration_tests.saas.connector_runner import (
+    ConnectorRunner,
+    generate_random_email,
+)
+from tests.ops.test_helpers.vault_client import get_secrets
+
+secrets = get_secrets("adyen")
+
+
+@pytest.fixture(scope="session")
+def adyen_secrets(saas_config) -> Dict[str, Any]:
+    return {
+        "domain_management": pydash.get(saas_config, "adyen.domain_management")
+        or secrets["domain_management"],
+        "domain_ca": pydash.get(saas_config, "adyen.domain_ca") or secrets["domain_ca"],
+        "api_key": pydash.get(saas_config, "adyen.api_key") or secrets["adyen.api_key"],
+        "merchant_account": pydash.get(saas_config, "adyen.merchant_account")
+        or secrets["adyen.merchant_account"],
```

Comment:
Added merchant account here. You'll need to change `merchantAccount` and `pspReference` in your `saas_config.toml` after this change